app/controllers/items/partial_dispatch.js

```diff
@@ -0,0 +1,53 @@
+import Ember from 'ember';
+import AjaxPromise from 'stock/utils/ajax-promise';
+const { getOwner } = Ember;
+
+
+export default Ember.Controller.extend({
+  item: null,
+  queryParams: ["orderPackageId"],
+  orderPackageId: null,
+  partialDispatchBackLinkpath: Ember.computed.localStorage(),
+
+  orderPackage: Ember.computed('orderPackageId', function(){
+    var orderPackage = this.get('store').peekRecord('OrdersPackage', this.get('orderPackageId'));
+    return orderPackage;
```

Comment:
Can we directly return **this.get('store').peekRecord('OrdersPackage', this.get('orderPackageId'));**? Is there any need for declaring variable?